src/python/WMComponent/TaskArchiver/TaskArchiverPoller.py

```diff
@@ -208,9 +217,8 @@ def completeTasks(self, finishedwfs):
 
         centralCouchAlive = True
         try:
-            abortedWorkflows = self.centralCouchDBWriter.getRequestByStatus(["aborted"])
-            logging.info("There are %d requests in 'aborted' status in central couch.", len(abortedWorkflows))
-
+            abortedWorkflows = self.killCondorJobsByWFStatus("aborted")
```

Comment:
I don't think that is a big difference given this will be polled every few minutes. But I will update that. :-)